docs/guides/arch.md

```
@@ -0,0 +1,394 @@
+
+If you own a ROG Ally or ROG Ally X ChimeraOS might be a good choice.
+
+## Content
+
+- [Introduction](#introduction)
+- [Installation](#installation)
+  - [Repository](#repository)
+  - [Asusctl](#asusctl)
+  - [ROG Control Center](#rog-control-center)
+  - [Graphics Switching](#graphics-switching)
+  - [Custom kernel - drivers fixes, hardware support](#custom-kernel-drivers-fixes-hardware-support)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)